pool/service/service.ml

```diff
@@ -61,8 +61,13 @@ module Email = struct
          else_fcn email new_recipient)
   ;;
 
-  let send ?ctx email =
+  let send ?sender ?ctx email =
     Logs.info (fun m -> m "Send email to %s" email.Sihl_email.recipient);
+    let email =
```

Comment:
extract to function and reuse in send and bulk_send

Reply:
Done